tests/suite-unit/PluginFlyvemdmAgent.php

```diff
@@ -243,4 +277,102 @@ public function testGetAgents() {
          $this->boolean(isset($agents[$agent->getID()]))->isTrue();
       }
    }
+
+   /**
+    * @tags testAddDefaultJoin
+    */
+   public function testAddDefaultJoin() {
+      $instance = $this->newTestedInstance();
+      $result = $instance::addDefaultJoin();
+      $this->string($result)->isEmpty();
+   }
+
+   /**
+    * @tags testAddDefaultWhere
+    */
+   public function testAddDefaultWhere() {
+      $instance = $this->newTestedInstance();
+      $result = $instance::addDefaultWhere();
+      $this->string($result)->isEmpty();
+   }
+
+   /**
+    * @tags testGetTopic
+    */
+   public function testGetTopic() {
+      $instance = $this->newTestedInstance();
+      $this->variable($instance->getTopic())->isNull();
+
+      $instance = $this->enrolTestAgent();;
+      $serial = $instance->getComputer()->getField('serial');
+      $entityId = $instance->getField('entities_id');
+      $this->string($instance->getTopic())->isEqualTo($entityId . '/agent/' . $serial);
+   }
+
+   /**
+    * @tags testGetPackages
+    */
+   public function testGetPackages() {
+      $instance = $this->newTestedInstance();
+      $this->array($instance->getPackages())->isEmpty();
+   }
+
+   /**
+    * @tags testGetFiles
+    */
+   public function testGetFiles() {
+      $instance = $this->newTestedInstance();
+      $this->array($instance->getFiles())->isEmpty();
+
+      $instance = $this->newTestedInstance();
+      $this->array($instance->getFiles())->isEmpty();
+   }
+
+   /**
+    * @tags testGetFleet
+    */
+   public function testGetFleet() {
+      $instance = $this->newTestedInstance();
+      $this->variable($instance->getFleet())->isNull();
+
+      $instance = $this->enrolTestAgent();
+      $this->object($instance->getFleet())->isInstanceOf('PluginFlyvemdmFleet');
+   }
+
+   /**
+    * @tags testGetComputer
+    */
+   public function testGetComputer() {
+      $instance = $this->newTestedInstance();
+      $this->variable($instance->getComputer())->isNull();
+   }
+
+   /**
+    * @tags testRefreshPersistedNotifications
+    */
+   public function testRefreshPersistedNotifications() {
+      $instance = $this->newTestedInstance();
+      $this->variable($instance->refreshPersistedNotifications())->isNull();
```

Comment:
add a TODO comment to tell the test is incomplete. I guess this will be complex to test the other cases right now

Reply:
done